test/acceptance/docker-compose.yaml

```diff
@@ -9,4 +9,9 @@ services:
       CHROME_ARGS: "--headless --ignore-certificate-errors"
       BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
       BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
-      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
\ No newline at end of file
+      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
+      JWT_SECRET: ${JWT_SECRET}
+      JWT_ISSUER: ${JWT_ISSUER}
+      NODE_TLS_REJECT_UNAUTHORIZED: "0"
+      NOTIFY_TEMPLATE_ID_FARMER_APPLY_LOGIN: ${NOTIFY_TEMPLATE_ID_FARMER_APPLY_LOGIN}
```

Comment:
i think we are fine to hard code this as it will not change and will replaced within weeks